### ISSUE TITLE
Quicktext doesn't fully load in Thunderbird new message window opened from URL when Thunderbird is not open

### ISSUE THREAD

**Issue by AdamBrezik** [thundernest/quicktext#402]
1. Have Thunderbird closed (the process isn't running).
2. Click on a mailto URL.

Expected: New message window opens with quicktext templates available.
Actual: Quicktext icon is present in the corner, but template categories are missing and the button itself doesn't react.

If you deem this a Thunderbird issue instead, just let me know and I'll track it there.

**Comment by jobisoft** [comment]
Fixed in Thunderbird Beta.

**Comment by jobisoft** [comment]
Confirmation that it is fixed for *you* as well would be awesome :-)

**Comment by AdamBrezik** [comment]
Sorry, I just assumed. I can't get it working, actually. I can see the extension in Thunderbird, however it says "(disabled)" as a postfix and I can't seem to enable it. Tried to find many a solution for this but I'm so far unsuccessful. I got Quicktext 5.14.

**Comment by jobisoft** [comment]
Ah, that s because QT is not compatible yet with Beta. Loop of doom. I fixed the *general* issue in Thunderbird Beta, but forgot that I also have to make QT compatible with Beta.

**Comment by matttbe** [comment]
Hello

Sorry to jump in the discussion here with a non directly related issue: @jobisoft I was going to suggest a modification of the manifest file to support the new Beta version, then I saw your last comment:

> QT is not compatible yet with Beta. Loop of doom. I fixed the general issue in Thunderbird Beta, but forgot that I also have to make QT compatible with Beta.

Is there something else that needs to be done to support the last Beta version? (I'm not an heavy user of QuickText, only using it to quickly insert 'Reviewed-by' tag in patch reviews -- but that's helpful, thank you for this add-on! :) -- maybe there are obscure options not working with the Beta?)

*EDIT*: I forgot to say: I modified the manifest in the `.xpi` file to accept `124.*` versions, and installed it. It worked for my usage. I can prepare a new PR with that (or without the 'max' limit) if you want me to.
